code/modules/hydroponics/grown/cannabis.dm

```diff
@@ -2,7 +2,6 @@
 /obj/item/seeds/cannabis
 	name = "pack of cannabis seeds"
 	desc = "Taxable."
-	icon = 'goon/icons/obj/hydroponics.dmi'
```

Comment:
No Remie, as I said in the PR itself aswell. I made new sprites for the seed packets for the cannibas plants. Said sprites are in the normal seeds sprites file. Heck, the seeds are even pictured.